### ISSUE TITLE
Repeater on FM and QSY Wiping conflict

### ISSUE THREAD

**Issue by vazquezrodjoel** [foldynl/QLog#503]
I like the QSY Wiping function on HF, but just notice that when using in on a repeater, like on 10m FM, if you type the callsign when you transmit to keep the QSO going, it conflicts.  For now I disabled that function.

**Comment by foldynl** [comment]
that's why you can define more profiles for the rig in which you can turn off/on the QSY Wiping function. That's the principle of the profiles in QLog.

And the upcoming version goes even further - there you will be able to define an Activity (combination of rig profile, antenna, rotator, and selected fields value)

**Comment by vazquezrodjoel** [comment]
OK, But does the profile get switched automatically based on the radio band/mode/frequency? I mean the current version...

**Comment by foldynl** [comment]
The rig is in the same state as it was before the change. That is, if it was On, then changing the profile will not change the state of Rig. 

In the new version, which will hopefully be very soon, it will be possible to define the status of Rig for individual profiles.

**Comment by vazquezrodjoel** [comment]
Ok, but my question is, is there a way to switch those profiles automatically based on say bands, or frequencies or something like that?  Or manual switch of the profile is the only option?

**Comment by Skittlebrau** [comment]
This is an issue when working split too.  Every time you transmit it wipes the QSO.  I just learned by reading this bug that you can disable the QSY wiping entirely, but it's a nice feature to have (when working non-split).  Unless I'm misunderstanding how the profile thing will work, I don't think I'd want to have to deal with different profiles for split vs. normal operation.

**Comment by vazquezrodjoel** [comment]
Exactly.  I think that, unless the rig profiles are changed automatically depending on different rules, it would be nice to consider like a buffer frequency difference depending the band.  I don't do much logging over 50mhz, but I know there are users that do, so for those I would suggest that the higher the frequency, the bigger that buffer could be.

Sometimes you are trying to work a station on split, and when you switch the radio to adjust the split frequency, the station gets cleared from the New QSO section, which is inconvenient, but I do like the QSY Wiping...  just my opinion.

**Comment by foldynl** [comment]
to be honest, it seems to me that you are solving a problem that is not there. If QSY Wipe is unwanted, you can turn it off.

Profiles currently do not support the functionality otherwise called "IF This Then That" because I think it would be incomprehensible to many operators. This does not mean that I cannot change my opinion, but I recently thought about something similar to Ant profile, and it seemed too complicated for a regular operator to use.

**Comment by Skittlebrau** [comment]
I like QSY wipe in general, but I only want it to work on the RECEIVE frequency, not the transmit one.  As is, the problem is that every time you transmit in split mode, QLog thinks you've QSYed (even though you are listening in the same spot the whole time), and the QSO is wiped.  I am not sure under what circumstances you'd want that behavior to be honest.

**Comment by vazquezrodjoel** [comment]
I think this is a great functionality of the software.  I used to use MacLoggerDX and when this function was implemented I loved it. because if I click on a station from the cluster and I could not copy that station I didn't had to clear anything, I just move on and it clears on its own.  That's why I was so excited founding it in here.  But I can't leave it active on any band as it is.  
If I'm on HF and need to work split, it gets on the way, if I'm listening for an opening to work a station and I hit the VFO by accident, it gets on the way.  If I'm working FT8 it sometimes keeps appearing and clearing (and querying the call logbook service) until the JTDX final log, depending on how JTDX handles the frequency.  I honestly think that, to be really useful, it need some adjustment.

**Comment by foldynl** [comment]
introduced split detection. that should help. I'm closing the issue
